pkg/sbproxy/reconcile/reconcile_brokers.go

```diff
@@ -61,7 +61,9 @@ func (r *resyncJob) reconcileBrokers(ctx context.Context, existingBrokers, desir
 			platformBroker, shouldBeProxified := brokerKeyMap[getBrokerKey(desiredBroker)]
 
 			if shouldBeProxified {
-				r.updateBrokerRegistration(ctx, platformBroker.GUID, desiredBroker)
+				if r.options.TakeoverEnabled {
```

Comment:
If you like the term takeover, could you unify the other alternative words used (such as proxifying, proxification, etc) to use takveOver as well

Reply:
Addressed.